### ISSUE TITLE
Has development moved to inside Unity?

### ISSUE THREAD

**Issue by mikeskydev** [atteneder/glTFast#650]
Hi, I've been keeping a track on development of the internal fork of glTFast Unity uses (https://docs.unity3d.com/Packages/com.unity.cloud.gltfast@latest)

I notice that there hasn't been much activity here, but the Unity fork has since released 5.1, 5.2 and 6.0.

Has development moved internally to Unity now, is this repo no longer the main source?

I would love to have some of the features listed in the Unity changelogs publicly available, especially the extensions API for #226

**Comment by atteneder** [comment]
Yes, development continued on an internal fork. We're working on getting those improvements into the hands of the public again (and also on this repository).
Feel free to subscribe to this issue and watch out for an update in the upcoming weeks.

**Comment by mikeskydev** [comment]
Thanks for the info! Is the Unity fork going to be the primary source going forward, and therefore is there any way to have PRs accepted now? While I understand work may progress faster internally, it seems a little odd to not be building in public like before, either in this repo or the existing public Unity fork.

**Comment by atteneder** [comment]
TL;DR: Yes, we'll keep on accepting PRs.

Once we roll things out, there will be a guide how to submit PRs, preferably targetting the Unity fork where changes will be synced to this repo and the OpenUPM release. Sorry for the delayed review on existing PRs.
As much as I'd love to keep things simple, considering all the internal and external requirements and limitations, setting up a good devops workflow is not straight forward.

**Comment by sambaas** [comment]
@atteneder We are considering writing support for some Cesium extentions. Would it be smart for us to wait for this reworked Unity fork to go live?
